Service/Microsoft.Health.DeIdentification.Web/RouteNames.cs

```diff
@@ -5,10 +5,12 @@
 
 namespace Microsoft.Health.DeIdentification.Web
 {
-    internal class RouteNames
+    internal static class RouteNames
```

Comment:
maybe we can remove this static class, and generate URL directly?

Reply:
I will modify it